ports/stm32/sdio.c

```diff
@@ -93,6 +93,21 @@ void sdio_deinit(void) {
     #endif
 }
 
+void sdio_reenable(void) {
+    if (__HAL_RCC_SDMMC1_IS_CLK_DISABLED()) {
+        __HAL_RCC_SDMMC1_CLK_ENABLE(); // enable SDIO peripheral
+        sdio_enable_high_speed_4bit();
```

Comment:
should this be here or called after `sdio_reenable()` in cyw43_ctrl.c ?

Reply:
I think keep it here (no functional change then).